app/frontend/components/tags/tag_assignment_menu.js

```diff
@@ -54,6 +54,12 @@ class TagAssignmentMenu {
     }
   }
 
+  hideTagAssignmentMenuAfterDelay() {
+    setTimeout(() => {
+      this.hideTagAssignmentMenu();
+    }, 500);
```

Comment:
@tobias-klein I think we can slightly increase it to 600ms

Reply:
Ok!